cmd/minishift/cmd/update.go

```diff
@@ -68,58 +74,26 @@ func runUpdate(cmd *cobra.Command, args []string) {
 		proxyConfig.ApplyToEnvironment()
 	}
 
-	localVersion, err := update.CurrentVersion()
+	currentVersion, err := update.CurrentVersion()
 	if err != nil {
 		atexit.ExitWithMessage(1, fmt.Sprintf("Update failed: %s", err))
 	}
-
-	latestVersion, err := update.LatestVersion()
-	if err != nil {
-		atexit.ExitWithMessage(1, fmt.Sprintf("Update failed: %s", err))
+	versionToUpdate, err := update.LatestVersion()
```

Comment:
@budhrg, why we are not handling the err here? I think it should be handled properly might be just after Line#84.